tests/actions.go

```diff
@@ -489,7 +491,6 @@ func (oa *operatorActions) InstallCRDOrDie() {
 	if err != nil {
 		glog.Fatalf("Failed to run '%s': %v", strings.Join(cmdArgs, " "), err)
 	}
-	oa.runKubectlOrDie("api-resources")
```

Comment:
not necessary, because other commends will try to fresh cache too, no need to call it.